frontend/store/companies/actions.js

```diff
@@ -0,0 +1,66 @@
+import types from "./mutation-types";
+
+export default {
+  async setCompanies({ commit }) {
```

Comment:
This functions should be named as getters, eg. `getCompanies`, because purpose of action is to get data, and purpose of mutation is to set them.

Reply:
Well that would mix them with getters, I can rename them to` fetchCompanies` etc.